src/main/java/seedu/address/ui/PinnedPersonCard.java

```diff
@@ -0,0 +1,83 @@
+package seedu.address.ui;
+
+import java.util.Comparator;
+import java.util.Optional;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.Label;
+import javafx.scene.layout.FlowPane;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.Region;
+import seedu.address.model.person.Birthday;
+import seedu.address.model.person.Person;
+
+/**
+ * An UI component that displays information of a {@code Person}.
+ */
+public class PinnedPersonCard extends UiPart<Region> {
+
+    private static final String FXML = "PinnedPersonListCard.fxml";
+
+    /**
+     * Note: Certain keywords such as "location" and "resources" are reserved keywords in JavaFX.
+     * As a consequence, UI elements' variable names cannot be set to such keywords
+     * or an exception will be thrown by JavaFX during runtime.
+     *
+     * @see <a href="https://github.com/se-edu/addressbook-level4/issues/336">The issue on AddressBook level 4</a>
+     */
+
+    public final Person person;
+
+    @FXML
+    private HBox cardPane;
+    @FXML
+    private Label name;
+    @FXML
+    private Label id;
+    @FXML
+    private Label phone;
+    @FXML
+    private Label address;
+    @FXML
+    private Label email;
+    @FXML
+    private FlowPane tags;
+    @FXML
+    private Label birthday;
+
+    /**
+     * Creates a {@code PersonCode} with the given {@code Person} and index to display.
+     */
+    public PinnedPersonCard(Person person, int displayedIndex) {
+        super(FXML);
+        this.person = person;
+        id.setText(displayedIndex + ". ");
+        name.setText(person.getName().fullName);
+        phone.setText(person.getPhone().value);
+        address.setText(person.getAddress().value);
+        email.setText(person.getEmail().value);
+        person.getTags().stream()
+                .sorted(Comparator.comparing(tag -> tag.tagName))
+                .forEach(tag -> tags.getChildren().add(new Label(tag.tagName)));
+        Optional<Birthday> possibleBirthday = person.getBirthday();
+        birthday.setText(possibleBirthday.map(Birthday::display).orElse(""));
+    }
```

Comment:
Am wondering if it would be better to reuse the UI for the PersonCard and only reorder the person to have pinned persons up top and maybe an additional symbol/ color changes rather than create a brand new pinned person card. What do you think?

Reply:
Oh ok sounds better! Ill change that the next iteration